### ISSUE TITLE
Narrator doesn't announce expanded information on Combobox

### ISSUE THREAD

**Issue by cnamal** [OfficeDev/office-ui-fabric-react#9869]
Similarly to #9659, Narrator doesn't convey the `aria-expanded` information for Combobox. 

Repro steps:
Go to https://developer.microsoft.com/en-us/fabric#/controls/web/combobox
Start Narrator
Tab until you get to the _ComboBox with toggleable freeform/auto-complete_ example combobox with both freeform and autocomplete disabled.
Narrator doesn't convey the `aria-expanded` information for Combobox. 

Impacted products: AI Builder in Powerapps

Notes:
- Works with NVDA and JAWS
- W3C provides a working Combobox:
https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html
I think the difference is W3C's expanded information is on a div, and on OUFR it's on an input

**Comment by joschect** [comment]
@cnamal Could you confirm the repro steps, I am able to hear that it says "collapsed" when it's highlighted, although it doesn't seem to announce when it's opened.

**Comment by cnamal** [comment]
@joschect  I can confirm that I do not hear "collapsed" when the combobox is highlighted. I have a video with the repro I can send you by email/Teams if need be :)
Please note that I didn't provide the correct URL in my repro steps, and I now corrected it. However, neither the dropdown (previous url) nor combobox appear to say the correct information.

**Comment by pandey11** [comment]
@joschet This is not working for my application as well although 'aria-expanded' is toggling properly.
@cnamal @joschect do we need to set aria-expanded text explicitly or is there any prerequisite to make it work

**Comment by jspurlin** [comment]
This is by design. I know a subset of ATs won't read the control as a combobox if it isn't set on the input.

For additional info, see the aria spec: https://www.w3.org/TR/wai-aria-1.1/#combobox , it states:
>The ARIA 1.0 specification describes a combobox pattern where a text input element has the combobox role and owns a listbox element. User agents, assistive technologies, and conformance checkers SHOULD continue to support the ARIA 1.0 pattern so that existing implementations of the ARIA 1.0 pattern remain functional.

**Comment by jspurlin** [comment]
@dzearing, If anything this is something that could be brought up with the Narrator team since they should be representing the content correctly
